src/de/jost_net/JVerein/gui/control/LehrgangControl.java

```diff
@@ -115,6 +119,7 @@ public void handleEvent(Event event)
         }
       }
     });
+    lehrgangsart.setMandatory(true);
```

Comment:
Warum ist "Von/Am" nicht auch ein Pflichtfeld? ohne macht das nicht viel Sinn.

Reply:
War bisher nicht implementiert. Finde ich aber auch sinnvoll. Dann muss es aber auch in den plausi Check.
Kann ich machen

Reply:
Ist gemacht